pkg/graph/schema.graphql

```diff
@@ -2253,6 +2288,9 @@ type Mutation {
   updateAccessibilityRequestCedarSystem(
     input: UpdateAccessibilityRequestCedarSystemInput
   ): UpdateAccessibilityRequestCedarSystemPayload
+  createSystemIntakeActionProgressToNewStep(
+    input: SystemIntakeProgressToNewStepsInput
+  ): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
```

Comment:
Similar comment to Lee's PR here https://github.com/CMSgov/easi-app/pull/2129#discussion_r1280671533

Though - if it makes sense (now seeing it in both places) for consistency, this is something I'm fine with just leaving as-is!

Reply:
I was thinking about that, wasn't sure which way to go; I ended up going with `UpdateSystemIntakePayload` for consistency with existing code.

Reply:
Totally fine with that. I'll resolve this and Lee's